tdms/include/simulation_parameters.h

```diff
@@ -1,15 +1,28 @@
 
+struct PML{
+  int Dxl = 0;  // Thickness of lower pml in the x direction
+  int Dxu = 0;  //              upper
+  int Dyl = 0;
+  int Dyu = 0;
+  int Dzl = 0;
+  int Dzu = 0;
```

Comment:
Can we have `PerfectlyMatchedLayer` ?
If you don't like the long name (I understand and don't complain), then please docstring.

Reply:
we can and we shall – the alignment disadvantage is minimal!